conf/openmetadata.yaml

```diff
@@ -223,6 +223,14 @@ slackEventPublishers:
 fernetConfiguration:
   fernetKey: ${FERNET_KEY:-jJ/9sz0g0OHxsfxOoSfdFdmk3ysNmPRnH3TUAbz3IHA=}
 
+secretsManagerConfiguration:
+  secretsManager: ${SECRET_MANAGER_CLASS_NAME:-LocalSecretsManager}
```

Comment:
We should be take name for the class with package ..like
org.openmetadata.catalog.secrets.LocalSecretsManager

Reply:
What do you think about creating an Enum for the different types of secret managers and following the same approach as the auth provider? 
```yaml
authProvider: ${AIRFLOW_AUTH_PROVIDER:-"no-auth"} # Possible values are "no-auth", "azure", "google", "okta", "auth0", "custom-oidc", "openmetadata"
```

Reply:
yup that could be useful in case we want to provide info on the type of secrets manage used , for ex in authProvider, it is needed because UI uses it to display login type images, so if we have requirement in code to identify it would be good to add it